lua/entities/prop_trash.lua

```diff
@@ -77,20 +77,22 @@ function ENT:SetupDataTables()
 end
 
 PropTrashExplosiveModels = {
-	["models/props_c17/oildrum001_explosive.mdl"]=true,
```

Comment:
In the future don't "fix" whitespace, it makes it harder to see what you actually changed

Reply:
will make sure to not do this in the future